test/Integration/SqlInputBindingIntegrationTests.cs

```diff
@@ -137,23 +137,31 @@ public async void GetProductNamesViewTest(SupportedLanguages lang)
         [Theory]
         [SqlInlineData("en-US")]
         [SqlInlineData("it-IT")]
-        [UnsupportedLanguages(SupportedLanguages.JavaScript, SupportedLanguages.PowerShell, SupportedLanguages.Java, SupportedLanguages.OutOfProc)] // IAsyncEnumerable is only available in C#
+        [UnsupportedLanguages(SupportedLanguages.JavaScript, SupportedLanguages.PowerShell, SupportedLanguages.Java)] // IAsyncEnumerable is only available in C#
         public async void GetProductsColumnTypesSerializationAsyncEnumerableTest(string culture, SupportedLanguages lang)
         {
             this.StartFunctionHost(nameof(GetProductsColumnTypesSerializationAsyncEnumerable), lang, true);
 
             string datetime = "2022-10-20 12:39:13.123";
+            ProductColumnTypes[] expectedResponse = new[]
+            {
+                new ProductColumnTypes()
+                {
+                    ProductID = 999,
+                    Datetime = DateTime.Parse(datetime),
+                    Datetime2 = DateTime.Parse(datetime)
+                }
+            };
             this.ExecuteNonQuery("INSERT INTO [dbo].[ProductsColumnTypes] VALUES (" +
                 "999, " + // ProductId
                 $"CONVERT(DATETIME, '{datetime}'), " + // Datetime field
                 $"CONVERT(DATETIME2, '{datetime}'))"); // Datetime2 field
 
             HttpResponseMessage response = await this.SendInputRequest("getproducts-columntypesserializationasyncenumerable", $"?culture={culture}");
             // We expect the datetime and datetime2 fields to be returned in UTC format
-            string expectedResponse = "[{\"productId\":999,\"datetime\":\"2022-10-20T12:39:13.123Z\",\"datetime2\":\"2022-10-20T12:39:13.123Z\"}]";
             string actualResponse = await response.Content.ReadAsStringAsync();
-
-            Assert.Equal(expectedResponse, TestUtils.CleanJsonString(actualResponse), StringComparer.OrdinalIgnoreCase);
+            ProductColumnTypes[] actualProductResponse = JsonConvert.DeserializeObject<ProductColumnTypes[]>(actualResponse);
+            Assert.Equal(expectedResponse, actualProductResponse);
```

Comment:
Can you explain what this change does? Why didn't `CleanJsonString()` work in this case?

Reply:
CleanJsonString method is added by me while working on tests for javascript/node, the javascript functions returned json with newlines and tabs in the response ex: [\r\n  {\r\n    \"ProductID\": 999,\r\n    \"Datetime\": \"2022-10-20T12:39:13.123\",\r\n    \"Datetime2\": \"2022-10-20T12:39:13.123\"\r\n  }\r\n] 
So added that method to clean the json to remove \r\n. 
Since this test is disabled for JavaScript and is failing for OOP because the actual response has casing issue. So updated it to deserialize into objects and compare.